vars/publishHelmChartArtifactory.groovy

```diff
@@ -22,21 +22,7 @@ def call(params) {
     sh "helm repo add ${it.name} ${it.url}"
   }
   sh "helm package --dependency-update --version ${VERSION} --app-version ${VERSION} ${chartPath}/${chartName}"
-  rtServer (
-     id: "liatrio-artifactory",
-     url: params.get("helmRepository", "https://artifactory.liatr.io/artifactory/"),
-     credentialsId: params.get("helmRepositoryCredentials", "openshift-token")
```

Comment:
Since this param is going away, it should be removed from the comment on line 10